src/__tests__/CityList.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
- [ ] Please fix this error as the previous reviewer requested: 
![image](https://github.com/mohamedgamaleldin999999/metrics-app/assets/91486791/fe3b58f6-0077-42a5-8ac8-f290583dbb8f)